client/src/app/core/services/keycloak/keycloak.service.ts

```diff
@@ -69,6 +69,24 @@ export class KeycloakService {
     return this.decodedToken()?.github_id;
   }
 
+  getUserGithubProfilePictureUrl() {
```

Comment:
nitpick: could you please add the return type to this functions, so we have proper typing when using them?